### ISSUE TITLE
The default bitmap representation of the dragged item in Drag & Drop UI not showing in XAML Islands

### ISSUE THREAD

**Issue by jasonwurzel** [windows-toolkit/Microsoft.Toolkit.Win32#123]
## I'm submitting a...
- Bug report (I searched for similar issues and did not find one)

## Current behavior
When dragging an item from a listview (please see link to minimal repro below), in pure uwp the default is to show a bitmap representation of the dragged item:

![2019-05-21 (2)](https://user-images.githubusercontent.com/1893675/58085601-54028f00-7bbd-11e9-9a27-d7914bf92e2e.png)

When hosting the same code in WindowsXamlHost, the default UI is not shown, only the changes in the cursor icon (please don't mind the wrong dpi scaling).

![2019-05-21 (3)](https://user-images.githubusercontent.com/1893675/58085591-4ea54480-7bbd-11e9-97ab-a88e99fc7e08.png)

When observing the Debug Output, the following three lines are written, every time I start a drag operation: 

```
onecore\com\combase\dcomrem\stdid.cxx(725)\combase.dll!00007FF9D7CD40EA: (caller: 00007FF9D790AB9B) ReturnHr(13) tid(26c4) 8001010E The application called an interface that was marshalled for a different thread.

Exception thrown at 0x00007FF9D639A839 (KernelBase.dll) in ControlPlaygroundXamlIsland.exe: 0x8001010E: Eine Schnittstelle, die für einen anderen Thread marshalled war, wurde von der Anwendung aufgerufen.

onecoreuap\shell\dataexchange\winrt\lib\visualhost.cpp(98)\DataExchange.dll!00007FF9B618D528: (caller: 00007FF9B618D902) LogHr(11) tid(26c4) 8001010E The application called an interface that was marshalled for a different thread.
```

## Expected behavior
drag and drop should behave like in pure uwp.

## Minimal reproduction of the problem with instructions

(sample code is taken from https://github.com/microsoft/Windows-universal-samples/tree/master/Samples/XamlDragAndDrop and wrapped into a WindowsXamlHost solution)

repro: 
https://github.com/jasonwurzel/XamlIslandRepo

## Environment

```
Windows 10 Build Number:
- Insider Build (build number: 10.0.18362)

App min and target version:
- Insider Build (10.0.18362)

Visual Studio 
- 2019 (version: 16.0.4)

```

**Comment by azchohfi** [comment]
@marb2000 is this a known issue?
@jasonwurzel have you tried fixing the DPI issue? Maybe it's there but outside the visible windows.

**Comment by jasonwurzel** [comment]
@azchohfi yeah, the fix for the dpi issue was rather easy (reading the docs helps...), but the problem remains

**Comment by azchohfi** [comment]
I just tried this with the new MyGet package, and on a newer version of Windows and it is working.
Could you please test this again on 18362?

**Comment by jasonwurzel** [comment]
sorry for the long pause. I just checked again and I still see the mentioned behavior. I started from scratch and created a new solution according to the new XamlIsland docs. 

https://github.com/jasonwurzel/XamlIslandDragAdorner

In the solution you will find 
- a Wpf App (WpfApp1) 
- a real UWP App (RealApp)
- a dummy UWP App as per the docs (DummyApp)
- a UWP class library containing the dragable UserControl (UwpClassLibrary1)

The UserControl (MyUserControl1) is used (via XamlIsland) in WpfApp1 and natively in RealApp. When starting RealApp and dragging the control the image is shown, when starting WpfApp1 and dragging the control the image is not shown. Same as in my initial bug description.

My current Windows version is 18363. dotnet core version 3.0 RC1
